script/github-actions/pe-cleanup.js

```diff
@@ -0,0 +1,59 @@
+const fs = require('fs');
+const core = require('@actions/core');
+const yaml = require('js-yaml');
+
+/* eslint-disable no-console */
+
+const daysSinceUpdate = dateUpdated => {
+  const diff = new Date() - new Date(dateUpdated);
+  return diff / (1000 * 60 * 60 * 24);
+};
+
+const deleteFiles = valuesFiles => {
+  core.exportVariable('FILES_TO_DELETE', true);
+  valuesFiles.forEach(file => {
+    try {
+      fs.unlinkSync(
+        `./manifests/apps/preview-environment/dev/environment-values/${file}`,
+      );
+      console.log(`${file} removed`);
+    } catch (error) {
+      console.log(error);
+    }
+  });
+};
+
+if (process.env.TRIGGERING_EVENT === 'delete') {
+  const valuesFiles = fs
+    .readdirSync('./manifests/apps/preview-environment/dev/environment-values/')
+    .filter(file =>
+      file.includes(
+        `${process.env.CURRENT_REPOSITORY}-${process.env.DELETED_BRANCH}`,
+      ),
+    );
+
+  if (valuesFiles.length > 0) {
+    deleteFiles(valuesFiles);
+  } else {
+    core.exportVariable('FILES_TO_DELETE', false);
+  }
```

Comment:
*Optional.  Since this code is duplicated in both the `delete` and `schedule` you could probably abstract this out into a shared function